### PR TITLE
📇 UI work on notifications

### PR DESCRIPTION
**Pull request by WRadoslaw**
- #4407 
- #4408 
- #4413

### CODE REVIEW

File: packages/atlas/src/components/_navigation/TopbarStudio/TopbarStudio.tsx [hunk 1]
```diff
@@ -49,6 +49,9 @@ export const TopbarStudio: FC<StudioTopbarProps> = ({ hideChannelInfo, isMembers
     setIsWorkspaceOpen(false)
   }
 
+  // todo: add logic after orion is done
+  const unseedMemberNotifications = 1
```
Comment (attemka) [inline packages/atlas/src/components/_navigation/TopbarStudio/TopbarStudio.tsx]: is it a typo?

File: packages/atlas/src/components/_notifications/NotificationTile/NotificationTile.tsx [hunk 2]
```diff
@@ -95,93 +101,136 @@ export const NotificationTile: FC<NotificationProps> = ({
...
-              <Text as="span" variant="h300" color="colorText">
-                {`${member.handle} `}
+      <StyledListItem
+        loading={loading}
+        read={read}
+        variant="compact"
+        className={className}
+        nodeStart={
+          member ? (
+            <NotifactionIcon avatarUrls={avatarUrls ?? []} iconType={getNotificationIcon(notification)} />
```
Comment (attemka) [inline packages/atlas/src/components/_notifications/NotificationTile/NotificationTile.tsx:128]:
```suggestion
            <NotificationIcon avatarUrls={avatarUrls ?? []} iconType={getNotificationIcon(notification)} />
```

Comment (attemka) [reply packages/atlas/src/components/_notifications/NotificationTile/NotificationTile.tsx:128]: note that it's passing  `[]` as a fallback option, which **might** brake the logic of the `Avatar component` and it's ternaries

File: packages/atlas/src/components/_notifications/NotificationsViewContent/NotificationsViewContent.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+      <GridItem colSpan={{ xxs: 12, md: 10, lg: 8 }} colStart={{ md: 2, lg: 3 }}>
+        <Header>
+          <BackActionWrapper>
+            <Button variant="tertiary" size="medium" icon={<SvgActionChevronL />} onClick={() => navigate(-1)} />
+          </BackActionWrapper>
+          <TitleContainer>
+            <Text as="h4" variant="h400">
+              {isMemberType ? 'Member' : 'Channel'} notifications
+            </Text>
+
```
Comment (attemka) [inline packages/atlas/src/components/_notifications/NotificationsViewContent/NotificationsViewContent.tsx]: extra line

File: packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx [hunk 4]
```diff
@@ -53,79 +71,102 @@ export const NotificationsWidget: FC<NotificationsWidgetProps> = ({ ...rest }) =
...
-          >
-            <Text as="span" variant="t100">
-              Go to notification center
-            </Text>
-          </StyledButton>
-        </Wrapper>
+                    },
+                    children: [
+                      <div key="single">
+                        {notifications.map((notification, idx) => (
```
Comment (attemka) [inline packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx]: probably needs a fallback similar to `NotificationsViewContent.tsx` for the notification with no id?

**Review by attemka — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/_notifications/NotificationTile/NotificationTile.tsx (hunk 2 above)
Comment (WRadoslaw) [reply packages/atlas/src/components/_notifications/NotificationTile/NotificationTile.tsx:128]: It shouldn't

File: packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.tsx [hunk 5]
```diff
@@ -97,7 +97,8 @@ export const TopbarViewer: FC = () => {
   }
 
   const topbarButtonLoading = isAuthenticating || membershipsLoading
-
+  // todo: add logic after orion is done
+  const unseedChannelNotifications = 2
```
Comment (attemka) [inline packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.tsx]: typo?

File: packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.tsx [hunk 6]
```diff
@@ -132,13 +133,14 @@ export const TopbarViewer: FC = () => {
...
                     <SignedButtonsWrapper>
-                      <NotificationsWidget trigger={<NotificationsButton />} />
+                      <NotificationsWidget type="member" trigger={<NotificationsButton />} />
                       {!mdMatch && !searchOpen && (
                         <StyledAvatar
                           size={40}
                           assetUrls={memberAvatarUrls}
                           loading={memberAvatarLoading}
                           onClick={handleDrawerToggle}
+                          badge={unseedChannelNotifications}
```
Comment (attemka) [inline packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.tsx]: typo?

File: packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.tsx [hunk 7]
```diff
@@ -147,6 +149,7 @@ export const TopbarViewer: FC = () => {
                           assetUrls={memberAvatarUrls}
                           onClick={handleDrawerToggle}
                           loading={memberAvatarLoading}
+                          badge={unseedChannelNotifications}
```
Comment (attemka) [inline packages/atlas/src/components/_navigation/TopbarViewer/TopbarViewer.tsx]: typo?

File: packages/atlas/src/components/_overlays/MemberDropdown/MemberDropdownNav.tsx [hunk 8]
```diff
@@ -254,6 +255,9 @@ type ListItemOptionsProps = {
   listItems: [ListItemProps, ListItemProps] | [ListItemProps]
 }
 const ListItemOptions: FC<ListItemOptionsProps> = ({ publisher, closeDropdown, listItems }) => {
+  // todo: add logic after orion is done
+  const unseedChannelNotifications = 2
+  const unseedMemberNotifications = 1
```
Comment (attemka) [inline packages/atlas/src/components/_overlays/MemberDropdown/MemberDropdownNav.tsx]: typos?

File: packages/atlas/src/components/_overlays/MemberDropdown/MemberDropdownNav.tsx [hunk 9]
```diff
@@ -265,6 +269,7 @@ const ListItemOptions: FC<ListItemOptionsProps> = ({ publisher, closeDropdown, l
           nodeStart={<IconWrapper icon={<SvgActionPlay />} />}
           label={atlasConfig.general.appName}
           to={absoluteRoutes.viewer.index()}
+          nodeEnd={<Badge data-badge={unseedMemberNotifications} />}
```
Comment (attemka) [inline packages/atlas/src/components/_overlays/MemberDropdown/MemberDropdownNav.tsx]: typo?

File: packages/atlas/src/components/_overlays/MemberDropdown/MemberDropdownNav.tsx [hunk 10]
```diff
@@ -273,6 +278,7 @@ const ListItemOptions: FC<ListItemOptionsProps> = ({ publisher, closeDropdown, l
             nodeStart={<IconWrapper icon={<SvgActionAddVideo />} />}
             label="Studio"
             to={absoluteRoutes.studio.index()}
+            nodeEnd={<Badge data-badge={unseedChannelNotifications} />}
```
Comment (attemka) [inline packages/atlas/src/components/_overlays/MemberDropdown/MemberDropdownNav.tsx]: typo?

**Review by attemka — CHANGES_REQUESTED** [review]

**Review by thesan — APPROVED** [review]

**Review by attemka — APPROVED** [review]

**Pull request merged**